### PR TITLE
Remove https references

### PR DESCRIPTION
**Pull request by schoims**

### CODE REVIEW

**Review by patilsnr — APPROVED** [review]
Optimistic sign off.

File: readme.md [hunk 1]
```diff
@@ -96,19 +96,18 @@ Note that you can configure your TLS protocol version and ciphers by following [
...
+|------------------------------------------------------------------------------------------------------------------|---------------------|---------------------|---------------------|---------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Authentication](https://docs.microsoft.com/azure/iot-hub/iot-hub-security-deployment)                     | :heavy_check_mark:  | :heavy_check_mark:* | :heavy_check_mark:  | :heavy_check_mark:* | Connect your device to IoT Hub securely with supported authentication, including private key, SASToken, X-509 Self Signed and X-509 CA Signed. </br> *IoT Hub only supports X-509 CA Signed over AMQP and MQTT at the moment.  X509-CA authentication over websocket and HTTPS are not supported. |
```
Comment (abhipsaMisra) [inline readme.md:101]:
```suggestion
| [Authentication](https://docs.microsoft.com/azure/iot-hub/iot-hub-security-deployment)                     | :heavy_check_mark:  | :heavy_check_mark:* | :heavy_check_mark:  | :heavy_check_mark:* | Connect your device to IoT Hub securely with supported authentication, including private key, SASToken, X-509 Self Signed and X-509 CA Signed. </br> *IoT Hub only supports X-509 CA Signed over AMQP and MQTT at the moment.  X509-CA authentication over websocket is not supported. |
```

File: readme.md [hunk 2]
```diff
@@ -96,19 +96,18 @@ Note that you can configure your TLS protocol version and ciphers by following [
...
+| [Direct Methods](https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-direct-methods)                 | :heavy_check_mark:  | :heavy_check_mark:  | :heavy_check_mark:  | :heavy_check_mark:  | IoT Hub gives you the ability to invoke direct methods on devices from the cloud.  The SDK supports handler for method specific and generic operation. |
+| [Upload file to Blob](https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-file-upload)               | :heavy_minus_sign:  | :heavy_minus_sign:  | :heavy_minus_sign:  | :heavy_minus_sign:  | The user can use the device client to retrieve a SAS URI from IoT Hub (to use for file uploads), upload to Azure Storage blob using IoT Hub provided credentials (using a supported client library), and then use the device client to notify IoT Hub that a file upload has completed.   File upload requires HTTPS connection, but can be initiated from client using any protocol for other operations. |
```
Comment (abhipsaMisra) [inline readme.md:106]:
```suggestion
| [Upload file to Blob](https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-file-upload)               | :heavy_minus_sign:  | :heavy_minus_sign:  | :heavy_minus_sign:  | :heavy_minus_sign:  | The user can use the device client to retrieve a SAS URI from IoT Hub (to use for file uploads), upload to Azure Storage blob using IoT Hub provided credentials (using a supported client library), and then use the device client to notify IoT Hub that a file upload has completed. **File upload requires HTTPS connection, but can be initiated from client using any protocol for other operations.** |
```

File: readme.md [hunk 3]
```diff
@@ -96,19 +96,18 @@ Note that you can configure your TLS protocol version and ciphers by following [
...
-| Devices multiplexing over single connection                                                                      | :heavy_minus_sign:  | :heavy_minus_sign:  | :heavy_check_mark:  | :heavy_check_mark:  | :heavy_check_mark:  |
-| Connection Pooling - Specifying number of connections                                                            | :heavy_minus_sign:  | :heavy_minus_sign:  | :heavy_check_mark:  | :heavy_check_mark:  | :heavy_check_mark:  |
-| [IoT Plug and Play][pnp-device-dev-guide]                                                                        | :heavy_check_mark:  | :heavy_check_mark:  | :heavy_check_mark:*  | :heavy_check_mark:*  | :heavy_minus_sign:  | IoT Plug and Play lets you build smart devices that advertise their capabilities to Azure IoT applications. IoT Plug and Play devices don't require manual configuration when a customer connects them to IoT Plug and Play-enabled applications. You can read more [here](https://docs.microsoft.com/azure/iot-pnp/overview-iot-plug-and-play). </br> *Note: AMQP support is mainly targeted for Edge-based scenarios. |
```
Comment (drwill-ms) [inline readme.md:111]: Why remove PnP references?

**Review by andyk-ms — APPROVED** [review]

File: readme.md (hunk 3 above)
Comment (schoims) [reply readme.md:111]: Added PnP references back

**Pull request merged**